clidocstool_test.go

```diff
@@ -227,6 +227,9 @@ format: "default|<id>[=<socket>|<key>[,<key>]]"`)
 func TestGenAllTree(t *testing.T) {
 	setup()
 	tmpdir := t.TempDir()
+	// tmpdir, err := os.MkdirTemp("", "clidocstool")
+	// require.NoError(t, err)
+	// t.Log(tmpdir)
```

Comment:
Looks like this is some debug code left behind

Reply:
ah 🤦🏻 used this to update the fixtures. Removed it.